Assets/Shopify/Unity/SDK/CartLineItems.cs

```diff
@@ -418,6 +425,19 @@ public class CartLineItems {
             return Delete(variantId);
         }
 
+        /// <summary>
+        /// Deletes all line items
+        /// </summary>
+        public void Reset() {
+            if (OnChange != null) {
```

Comment:
Should this really wrap the whole thing? I would have thought this would always run the removal, and then the condition would be around OnChange call?

Reply:
Agreed - we should extract out the actual removal from the callback triggering :+1: